docs/installation/installation-allenact.md

```diff
@@ -16,23 +9,56 @@ PyTorch that supports your CUDA version. In such a case, after installing the be
 should follow the directions for installing PyTorch with older
 versions of CUDA available on the [PyTorch homepage](https://pytorch.org/).
 
-In order to install requirements we recommend creating a new
+In order to install `allenact` and/or its requirements we recommend creating a new
 [python virtual environment](https://docs.python.org/3/tutorial/venv.html) and installing all
-of the below requirements into this virtual environment. Several tools exist to help manage
-virtual environments, we have had success in using [Anaconda](https://docs.conda.io) and
-[`pipenv`](https://pipenv.kennethreitz.org/en/latest/)
+of the below requirements into this virtual environment.
 
 ### Installing requirements with `pip`
 
-Note: *do not* run the following if you have already installed requirements with `pipenv`
-as above. If you prefer managing your environment manually, all requirements may be installed using
- `pip` by running the following command:
+There are three main installation paths regarding whether you are interested in installing
+
+1. Just the main `allenact` framework (without any plugin).
+1. The main framework and one or more of the available plugins.
+1. The entire library, e.g. for development of more advanced features like a new training algorithm.
+
+#### Standalone framework
+
+```bash
+pip install allenact
```

Comment:
👍  we'll need to get allenact on pypi before this works of course. Maybe also add a section about pip installing from GitHub for those people who want to install the bleeding edge version on master without cloning the repo locally.

Reply:
Sure. I also realized I had a missing dependency for box2d (gym_plugin). I'll also add it to this PR.